docs/api-docs/analytics/BODL.md

```diff
@@ -0,0 +1,432 @@
+# Big Open Data Layer (BODL)
+
+<div class="otp" id="no-index">
+
+### On this page
+- [Overview](#overview)
+- [Pixel example snippets](#pixel-example-snippets)
+</div> 
+
+## Overview
+The Big Open Data Layer (shortened as BODL, pronounced 'Bottle') is a standard for obtaining data points necessary to drive storefront analytics and personalization. By moving all BigCommerce and headless storefronts to use this standard over time, merchants will benefit from increased site speed and more consistent third-party analytics integrations.
+
+## Pixel example snippets
+Below is a series of example snippets of code to build on when using BODL data for your integration.
+
+### Initialize script
+This larger code snippet has a lot of detection built-in to identify where it is running within the environment; then, the code injects objects based on this detection.
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--">
+<div class="HubBlock-content">
+  
+ <!-- theme:  -->
+
+The check for BODL is fetched once per page render instead of once for each analytics integration. If you want to make an alternate custom version of this data layer object with a unique schema, please use a unique object name. A unique object name ensures other apps depending on this default BODL schema do not break. We recommend using BODL_YOUR_APP_NAME, so it's unique and easy to determine its purpose.
```

Comment:
```suggestion
The check for BODL is fetched once per page render instead of once for each analytics integration. If you want to make an alternate custom version of this data layer object with a unique schema, please use a unique object name. A unique object name ensures other apps depending on this default BODL schema do not break. We recommend using BODL_YOUR_APP_NAME as the object name so that it's unique and easy to understand.
```